pt/news/_posts/2017-04-01-support-of-ruby-2-1-has-ended.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: news_post
+title: "Support of Ruby 2.1 has ended"
+author: "usa"
+translator: "fpgentil"
+date: 2017-04-01 00:00:00 +0000
+lang: pt
+---
+
+Anunciamos que o suporte da série Ruby 2.1 foi encerrado.
+
+Após essa _release_ do Ruby 2.1.10 no final de março do últmo ano,
+o suporte da série do Ruby 2.1 estava em fase de manutenção de segurança.
+Agora, após um ano, essa fase se encerrou. Contudo, em 31 de março de
+2017, todo o suporte da série do Ruby 2.1 foi encerrado. _Bugs_ e correções de
+seguranças de versões mais recentes do Ruby não serão mais trazidas à versão 2.1
+e nenhum _patch release_ do Ruby 2.1 será lançado. Nós recomendamos que você
+atualize para o Ruby 2.1 ou 2.3 o mais rápido possível.
```

Comment:
No original em inglês: We highly recommend that you upgrade to Ruby **2.4** or 2.3 as soon as possible
Você colocou 2.1 ou 2.3